datastore/unit_tests/test_batch.py

```diff
@@ -369,30 +369,62 @@ def begin(self):
         self.assertEqual(client._batches, [])
 
 
-class _PathElementPB(object):
+class Test__parse_commit_response(unittest.TestCase):
 
-    def __init__(self, id_):
-        self.id = id_
+    def _call_fut(self, commit_response_pb):
+        from google.cloud.datastore.batch import _parse_commit_response
 
+        return _parse_commit_response(commit_response_pb)
 
-class _KeyPB(object):
+    def test_it(self):
+        from google.cloud.grpc.datastore.v1 import datastore_pb2
+        from google.cloud.grpc.datastore.v1 import entity_pb2
 
-    def __init__(self, id_):
-        self.path = [_PathElementPB(id_)]
+        index_updates = 1337
+        keys = [
+            entity_pb2.Key(
+                path=[
+                    entity_pb2.Key.PathElement(
+                        kind='Foo',
+                        id=1234,
+                    ),
+                ],
+            ),
+            entity_pb2.Key(
+                path=[
+                    entity_pb2.Key.PathElement(
+                        kind='Bar',
+                        name='baz',
+                    ),
+                ],
+            ),
+        ]
+        response = datastore_pb2.CommitResponse(
+            mutation_results=[
+                datastore_pb2.MutationResult(key=key) for key in keys
```

Comment:
I'm surprised key/keys aren't reserved words, but it doesn't look like they are.
https://docs.python.org/2.5/ref/keywords.html